src/utils/constants.ts

```diff
@@ -11,6 +11,7 @@ export const MAX_TICKER_LENGTH = 12;
 export const MAX_MODULE_LENGTH = 32;
 export const MAX_MEMO_LENGTH = 32;
 export const MAX_OFF_CHAIN_METADATA_LENGTH = 32;
+export const MAX_BATCH_SIZE_SUPPORTING_SUBSIDY = 5;
```

Comment:
This value needs to be increased upto 7 from the core side in order to support `createNftCollection`